client/src/components/ProjectWizard/SidebarPoints/SidebarPoints.js

```diff
@@ -45,16 +53,30 @@ const useStyles = createUseStyles({
 
 const SidebarPoints = props => {
   const classes = useStyles();
-  const { rule } = props;
+  const { rule, rulesConfig } = props;
+
   const opacityTest =
     rule.value && rule.value !== "0" ? "" : classes.lowOpacity;
   const noToolTip = rule.value === 0 ? classes.noDisplay : "";
 
+  let target = rulesConfig.target.value;
+  let earned = rulesConfig.earned.value;
+
+  let earnedPointsColor =
+    (rule.name === "Earned Points" && earned === 0) ||
+    (rule.name === "Earned Points" && earned < target)
+      ? classes.ruleValueEarned
+      : classes.ruleValue;
```

Comment:
So just fyi, the reason I ended up writing the above comment about the theme colors was because I didn't know what "ruleValue" was and what color that was supposed to be when reading this line. Maybe we can also rename this class to make it more descriptive. (Though I can't think of one right now.. so if you can't either then it's okay.)